rareapi/fixtures/posttags.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "model": "rareapi.posttag",
+        "pk": 1,
+        "fields": {
+            "post_id": 1,
+            "tag_id": 1
+        }
+    },
+    {
+        "model": "rareapi.posttag",
+        "pk": 2,
+        "fields": {
+            "post_id": 1,
+            "tag_id": 1
+        }
```

Comment:
this fixture is the same as the first with the same post and tag ids. It isn't a huge deal right now but I could see it becoming an issue later!